configure.ac

```diff
@@ -153,6 +153,13 @@ AS_IF([test "x$enable_tar_selinux" = "xyes"],
 	    [TARSELINUX=no]
 )
 
+AC_ARG_ENABLE(
+        [stateless],
+        AS_HELP_STRING([--disable-stateless],[OS is not stateless, do not ignore configuration files (stateless by default)]),
+        AC_DEFINE(OS_IS_STATELESS,0,[OS is not stateless]),
+        AC_DEFINE(OS_IS_STATELESS,1,[OS is stateless])
+)
```

Comment:
There is one case not captured by the above macro: If `--enable-stateless` is passed, `OS_IS_STATELESS` will be set to 0.

Granted, one could hope that users will not pass options undocumented in `./configure --help`, but some people may want to always explicitly enable/disable all package-specific options.